test/wallet/get-wallet-details-batch.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Chance = require('chance');
+const _ = require('lodash');
+
+const Lemonway = require('../../');
+
+const chance = new Chance();
+
+describe('getWalletDetailsBatch', function () {
+  this.timeout(2000000);
+
+  it('should get a batch of wallet details', function (done) {
+    const lemonway = new Lemonway(process.env.LOGIN, process.env.PASS, process.env.ENDPOINT, process.env.WK_URL, { proxy: 'https://proxy.october.eu/lemonway/staging', namespaceArrayElements: false});
+    const prefix = chance.integer({ min: 10, max: 99 });
+    const walletIds = _.range(5).map(() => ({
+      wallet: `${prefix}${chance.integer({min: 100, max: 999 })}`,
```

Comment:
Don't we have a linter in here 😱 
```suggestion
      wallet: `${prefix}${chance.integer({ min: 100, max: 999 })}`,
```